pages/2017/index.rst

```diff
@@ -8,4 +8,60 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
```

Comment:
Same comment I made in chat - think 'on the' doesn't sound right somehow. I would say either 'from 8-12 May 2017' or 'on 8-12 May 2017'?